### PR TITLE
Docker for React Native Website

### PR DESCRIPTION
**Pull request by VISWESWARAN1998**
Added Dockerfile for easy deployment.

Thanks and Regards,
Visweswaran N

### CODE REVIEW

**Comment by VISWESWARAN1998** [comment]
I have considered adding a docker file because anyone with docker can easily deploy and test the website without any tools and installation procedures.

**Comment by rachelnabors** [comment]
That seems really cool, but I'm not sure that's a use case we're ready to support. Could you submit an update to the Repo's README to explain how to use as well?

**Comment by VISWESWARAN1998** [comment]
http://react-native-website.visweswaran.com is deployed using docker for your reference.

**Review by Simek — CHANGES_REQUESTED** [review]
Hey @VISWESWARAN1998, thank you for proposing this change.

Can you make sure that the `Dockerfile` is correct after the website has been migrated to the new framework? And please, can you remove the comment in this file?

Also it is not clear enough for me what are the benefits of offering and using the website with Docker? Can you elaborate on that?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node
+
+COPY . /rn_website
+WORKDIR /rn_website/website
+
+RUN npm install
```
Comment (Simek) [inline Dockerfile:6]: It is possible that `npm` command won't be working correctly after the introduction of Yarn workspaces in the repository. Can you test this out against current `master`?

**Comment by rachelnabors** [comment]
Going to close because Docker support seems like it is out of scope for this project

**Pull request closed without merging**